src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml

```diff
@@ -1,65 +1,69 @@
-                           Grid.Column="0" />
-                </Grid>
-                <Button x:Name="CloseButton"
+﻿<ContentPage
+    x:Class="ArcGISRuntime.Samples.DisplayUtilityNetworkContainer.DisplayUtilityNetworkContainer"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:cmp="clr-namespace:Microsoft.Maui.Controls.Compatibility;assembly=Microsoft.Maui.Controls"
+    xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui"
+    xmlns:resources="clr-namespace:ArcGISRuntimeMaui.Resources">
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esriUI:MapView
+            x:Name="MyMapView"
+            GeoViewTapped="MyMapView_GeoViewTapped"
+            Style="{DynamicResource EsriSampleGeoView}" />
+        <Border MinimumHeightRequest="200" Style="{DynamicResource EsriSampleControlPanel}">
+            <Grid>
+                <StackLayout Spacing="5">
+                    <Label Text="Tap an element to view its container." />
+                    <Label FontAttributes="Bold" Text="Utility association types" />
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="auto" />
+                            <RowDefinition Height="auto" />
+                            <RowDefinition Height="auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="25" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
+                        <Label
+                            Grid.Row="0"
+                            Grid.Column="1"
+                            Text="Attachment" />
+                        <Label
+                            Grid.Row="1"
+                            Grid.Column="1"
+                            Text="Connectivity" />
+                        <Label
+                            Grid.Row="2"
+                            Grid.Column="1"
+                            Text="Containment" />
+                        <Image
+                            x:Name="AttachmentImage"
+                            Grid.Row="0"
+                            Grid.Column="0" />
+                        <Image
+                            x:Name="ConnectivityImage"
+                            Grid.Row="1"
+                            Grid.Column="0" />
+                        <Image
+                            x:Name="ContainmentImage"
+                            Grid.Row="2"
+                            Grid.Column="0" />
+                    </Grid>
+                    <Button
+                        x:Name="CloseButton"
                         Margin="5"
                         Clicked="CloseButton_Clicked"
                         IsVisible="false"
```

Comment:
I think it might be a nicer user experience to enable/disable the button rather than showing/hiding.